### PR TITLE
fix: ButtonTemplateBuilder title ignores "0"

### PR DESCRIPTION
**Pull request by snmatsui**
close: #285

### CODE REVIEW

File: src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php [hunk 1]
```diff
@@ -113,8 +113,8 @@ public function buildTemplate()
             'actions' => $actions,
         ];
 
-        if ($this->title) {
-            $this->template['title'] = $this->title;
+        if (((string)($this->title)) !== "") {
+            $this->template['title'] = (string)($this->title);
```
Comment (Vaduz) [inline src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php:117]: Please use `isset` instead.
```suggestion
        if (isset($this->title)) {
            $this->template['title'] = $this->title;
```
https://www.php.net/manual/en/types.comparisons.php

**Comment by moririnson** [comment]
i'll merge this. @snmatsui thanks for your contribution!

File: src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php (hunk 1 above)
Comment (snmatsui) [reply src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php]: Thank you for the suggestion.
However, when `$this->title = ""`, title included the template.
Message including empty text is rejected from MessagingAPI.
```
{
    "message": "A message (messages[0]) in the request body is invalid",
    "details": [
        {
            "message": "must be non-empty text",
            "property": "template/columns/0/title"
        }
    ]
}
```
It looks good to exclude the case `$this->title = ""`.

Comment (Vaduz) [reply src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php]: Basically empty strings should be treated as set.
And we don't implement server side validation on the SDK include length of the strings, data structure and so on.
But we should implement input data handling consistently though. (Some class use `empty`)

**Pull request merged**